.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+# This script is executed whenever the docker container is (re)started.
+
+#===============================================================================
+# debugging
+set -x
+
+#===============================================================================
+# environment
+reentry scan
+
+export SHELL=/bin/bash
+
+#===============================================================================
+# setup AiiDA
+aiida_backend=django
+
+if [ ! -d /home/$SYSTEM_USER/.aiida ]; then
+    verdi setup                                \
+        --profile default                      \
+        --non-interactive                      \
+        --email some.body@xyz.com              \
+        --first-name Some                      \
+        --last-name Body                       \
+        --institution XYZ                      \
+        --db-backend $aiida_backend            \
+        --db-username aiida                    \
+        --db-password aiida_db_passwd          \
+        --db-name aiidadb                      \
+        --db-host localhost                    \
+        --db-port 5432                         \
+        --repository /home/$SYSTEM_USER/aiida_repository
+
+   echo 'eval "$(verdi completioncommand)"' >> /home/$SYSTEM_USER/.bashrc
```

Comment:
Maybe better to replace this with:
```
echo 'eval "$(_VERDI_COMPLETE=source verdi)"' >> /home/$SYSTEM_USER/.bashrc
```
since at some point we wanted to get rid of `verdi completioncommand`

Reply:
ok